src/focus-ring.js

```diff
@@ -36,7 +30,19 @@ document.addEventListener('DOMContentLoaded', function() {
    * @return {boolean}
    */
   function focusTriggersKeyboardModality(el) {
-    return matches(el, keyboardModalityWhitelist) && matches(el, ':not([readonly])');
+    var type = el.type;
+    var tagName = el.tagName.toLowerCase();
```

Comment:
You’re only testing `tagName` against `'input'` and `'textarea'`. Therefore, you could safely omit `.toLowerCase()` and test for `'INPUT'` and `'TEXTAREA'` instead.